templates/base/head.tmpl

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="{{.Language}}">
+<html lang="{{.Language}}" class="theme-{{.SignedUser.Theme}}">
```

Comment:
How to handle non-login user?

Reply:
I don't think anonymous users have access to code edditor

Reply:
he's right, condition should be improved as even unsigned users should have a theme, just not in this variable.